### DISCUSSION TITLE
Tutorial 2: Tmux, Btop and Htop

### DISCUSSION THREAD

**Discussion by nyameko** [chpc-tech-eval/scc#95] [Show and tell]
Your output should be similar to the following, when working with multiple panes and installing programs across different severs.

![tmux_btop_htop](https://github.com/chpc-tech-eval/chpc24-scc-nmu/assets/21206557/361e50a0-ee57-4d68-98b5-e2b2e65204ca)

![tmux_ssh_run_tops](https://github.com/chpc-tech-eval/chpc24-scc-nmu/assets/21206557/2140b8ca-baa4-42e2-850a-b8318239e5e1)

![tmux_running_htop_btop](https://github.com/chpc-tech-eval/chpc24-scc-nmu/assets/21206557/719f1890-0090-404c-bce4-cc24645fd92d)

**Comment by JCheney20** [comment]
**HPC_Wizards**
![image](https://github.com/chpc-tech-eval/chpc24-scc-nmu/assets/64533742/5a151765-0003-4441-bbb7-5cd4c1112205)

**Reply by n-zwane** [reply]
Team Technocrats SCC 2025
<img width="1920" height="1080" alt="tmux and btop" src="https://github.com/user-attachments/assets/40acf006-9812-46f0-9131-516514b4946e" />

**Comment by Talhahpatelia** [comment]
### Wits-A
## Btop on head node:
<img width="960" alt="headnode btop" src="https://github.com/chpc-tech-eval/chpc24-scc-nmu/assets/29210701/da7d143e-a9b3-4757-b930-95e607e0a322">

## Htop on compute node:
<img width="960" alt="compute node htop" src="https://github.com/chpc-tech-eval/chpc24-scc-nmu/assets/29210701/f3910b0c-e68d-4170-93bc-e80797468c4c">

## Tmux: head node on top and compute node on bottom:
<img width="960" alt="tmux" src="https://github.com/chpc-tech-eval/chpc24-scc-nmu/assets/29210701/52b5e7fb-128d-4bb5-a0e3-ff2324127bb2">

## htop and btop image:
btop on compute node on top and htop with head node on bottom. We had to resize the windows because of the btop contraints to minimum pane size with our screen resolution.

<img width="960" alt="both tmux" src="https://github.com/chpc-tech-eval/chpc24-scc-nmu/assets/29210701/eba189c9-ea74-4739-acfa-5f6e3198bce9">

**Comment by CraftierLamb599** [comment]
# WIts-1
![image](https://github.com/chpc-tech-eval/chpc24-scc-nmu/assets/137203651/a8bfd43f-aa5e-468e-a5c5-be90948c86d4)

**Comment by siyabonga-mashilwane** [comment]
ThunderBytes Split terminals (Tmux session)
![tutorial2 1](https://github.com/chpc-tech-eval/chpc24-scc-nmu/assets/81867821/ef348726-616d-4eea-9312-0b35dd5b0f28)

**Comment by siyabonga-mashilwane** [comment]
### **ThunderBytes**
![Screenshot 2024-07-09 161351](https://github.com/chpc-tech-eval/chpc24-scc-nmu/assets/81867821/9556fa42-1242-4b66-90d7-8787fc14484f)

**Comment by ThabangMmusi** [comment]
**Parallel Pioneers**

tmux screenshot

![PP Screenshots](https://github.com/chpc-tech-eval/chpc24-scc-nmu/assets/100237631/0bafc433-c050-491c-9c0d-4e7b9415bd17)

**Comment by Cyphers-UFH** [comment]
Cyphers - UFH

![Screenshot 2024-07-09 190549](https://github.com/chpc-tech-eval/chpc24-scc-nmu/assets/175048513/da1c5a41-859d-4e4d-a8cf-26e7b514a558)

**Comment by Dc-Spears** [comment]
**CPUT! (DC Spears)**

_tmux_
![cput-tmux-recent](https://github.com/chpc-tech-eval/chpc24-scc-nmu/assets/175037009/c55b3dbc-87f0-48fd-bee3-4b1597f1c603)

_btop && htop_
![cput-btop](https://github.com/chpc-tech-eval/chpc24-scc-nmu/assets/175037009/0f2ee520-6669-4958-ba68-91c9ebddf1e4)
![cput-htop](https://github.com/chpc-tech-eval/chpc24-scc-nmu/assets/175037009/cf6d9ac7-86b3-41e1-aa9a-cb96851997bc)

**Comment by Sphiwencube** [comment]
**### TOMORROWS LEGENDS (SMU)**

**btop & htop**

![IMG_20240710_141506](https://github.com/chpc-tech-eval/chpc24-scc-nmu/assets/169956748/156af2ec-820e-4d89-b51a-c4e76d69dbba)
![IMG_20240710_141550](https://github.com/chpc-tech-eval/chpc24-scc-nmu/assets/169956748/cafcb1b0-e017-4776-a98d-ca85dfb68c9c)

**Comment by SfisoMncube** [comment]
**Khulisa IP-A**

tmux
![Screenshot 2024-07-10 003534](https://github.com/chpc-tech-eval/chpc24-scc-nmu/assets/151764841/6ad360e6-5e3f-43a3-bb79-13348252538c)

btop and htop
![Screenshot 2024-07-10 004354](https://github.com/chpc-tech-eval/chpc24-scc-nmu/assets/151764841/295c1fbf-b1c4-4b47-a846-06102fa70689)
![Screenshot 2024-07-10 004535](https://github.com/chpc-tech-eval/chpc24-scc-nmu/assets/151764841/5482afa8-ef17-4809-8021-e88f530d91ed)

**Comment by 15kay** [comment]
HYPERNET

BTOP AND HTOP
![image](https://github.com/chpc-tech-eval/chpc24-scc-nmu/assets/94864138/c1d31e22-1061-42bf-b931-b22c54fefe81)
![image](https://github.com/chpc-tech-eval/chpc24-scc-nmu/assets/94864138/abdc0049-fe59-4655-9ddc-ad0d1e3418e7)

**Comment by ADELAIDEMOGALE** [comment]
### **Khulisa Data**

**tmux: working with multiple panes and installing programs across different severs.**

![Screenshot (680)](https://github.com/chpc-tech-eval/chpc24-scc-nmu/assets/166052302/f4eee71e-0730-433e-bdd6-f9d73440b081)

**btop on compute node(above our head node) and htop on head node(below our compute node)**

![Screenshot (688)](https://github.com/chpc-tech-eval/chpc24-scc-nmu/assets/166052302/708308fa-6688-4f5a-9d2d-91e6464facb4)

**Comment by MishkaSingh1** [comment]
**EEElite (UCT)**

_btop_
![image](https://github.com/chpc-tech-eval/chpc24-scc-nmu/assets/175049735/05b0e2ba-353e-4080-9af9-4beab6793491)

_tmux split screen with htop running on computenode1_
![image](https://github.com/chpc-tech-eval/chpc24-scc-nmu/assets/175049735/e1a5d3ba-9fb7-4bd3-ac2b-1537747b4b67)

**Comment by SizweMarole-21389** [comment]
btop, because it is easy for a first time user to navigate and understand.

![argument](https://github.com/chpc-tech-eval/chpc24-scc-nmu/assets/175023186/71dbd36a-9606-48f7-ae29-739746c630dc)

**Comment by prince-taker** [comment]
Ctrl Alt Elite
![image](https://github.com/user-attachments/assets/6bedb93c-4b6a-4282-9f2b-3446f78e7594)

**Comment by 15kay** [comment]
hypernet 
![image](https://github.com/user-attachments/assets/83f3c09e-0f18-4429-8f8b-a5b35f8449ae)

**Comment by 15kay** [comment]
hypernet
![image](https://github.com/user-attachments/assets/35fae13d-30da-4b6d-98de-98200043e12f)

**Comment by ndumezulu-ngwenya** [comment]
UNISA-SCT'24

Btop
![Screenshot 2024-07-12 130417](https://github.com/user-attachments/assets/45f26cd7-a4ce-46fd-9fe3-632ad324443b)

**Comment by Kohzmik** [comment]
CUT_Team
![image](https://github.com/user-attachments/assets/1381cd86-951c-49b0-a3c8-cdce18ffc359)

**Comment by KayGee-17** [comment]
Future Innovators
<img width="955" alt="btop" src="https://github.com/user-attachments/assets/3e81203a-1d75-4a95-a3bf-108ee293c7fe">
<img width="958" alt="htop" src="https://github.com/user-attachments/assets/c64c91e1-7e3c-432e-8eba-4b31e7afd2fb">

**Comment by JedK721** [comment]
**Wits-i**

btop and htop
![Screenshot 2024-07-13 163701](https://github.com/user-attachments/assets/be3b9a2a-b781-44d6-8da9-d317bef1606c)
![Screenshot 2024-07-13 163549](https://github.com/user-attachments/assets/ae71b3bc-60c4-4b89-b162-12dc13fe862b)

**Comment by 15kay** [comment]
[image: image.png][image: image.png]

BTOP HTOP

**Comment by KamvaMj** [comment]
DC Daggers

![Screenshot (72)](https://github.com/user-attachments/assets/c1598e6b-94f4-4228-a9f3-f908d6ac9d8e)

**Comment by Cloud-Strive** [comment]
Just a test....Hi

**Comment by DCspearsv2** [comment]
DC Spears btop and htop
![Screenshot 2025-07-08 165349](https://github.com/user-attachments/assets/0c646f57-71e7-47df-aa62-2ff65f0306fb)

**Comment by ShavirV** [comment]
TuksByters
![Screenshot 2025-07-08 172758](https://github.com/user-attachments/assets/00c72e53-f345-4efb-8538-0b03ca45e362)

**Comment by ruvarashecodes** [comment]
![Screenshot 2025-07-08 152022](https://github.com/user-attachments/assets/c266b6fc-227f-46f2-81e6-c52a230bbeab)
![Screenshot 2025-07-08 151859](https://github.com/user-attachments/assets/3a69c515-4464-416f-a11b-36898a5027fd)

#CtrlAltDefeat

**Comment by OddBitLoaders1** [comment]
OddBitLoaders
![Screenshot 2025-07-08 202334](https://github.com/user-attachments/assets/dc7bf092-7f66-4774-83d6-29b355449572)

**Comment by Abahle947** [comment]
Techtribe Btop and htop
![thumbnail_PHOTO-2025-07-08-20-20-17](https://github.com/user-attachments/assets/6222d2a8-7fb6-4a7e-b058-8a5eca22edd4)


![image1](https://github.com/user-attachments/assets/8d16bfa5-cf7c-4754-852a-6a84e3135552)

**Comment by Moagi99** [comment]
TUT_X

htop
![Screenshot (9)](https://github.com/user-attachments/assets/4ddb9efa-cffb-4ddd-bbea-22c18e33038d)
btop
![Screenshot (10)](https://github.com/user-attachments/assets/69edb826-4bd4-400b-8e22-b1666445fc54)

**Comment by UNISA-SCT25** [comment]
![top and htop results](https://github.com/user-attachments/assets/2a60ee89-4d24-4796-aef2-d73f87da1b55)

**Comment by MphephuSamuel** [comment]
computer champions
![image](https://github.com/user-attachments/assets/064ccf7d-ca75-4ff3-bcec-c8381ab759cd)
![image](https://github.com/user-attachments/assets/fb7c5ecb-9c19-4007-a937-8d01731d7e0c)

**Comment by Swissisking21** [comment]
Techno geeks
![Screenshot (273)](https://github.com/user-attachments/assets/8600ea98-6a7b-408c-8668-fff47c227336)

**Comment by Mwelase-bit** [comment]
DebugThugs
![htop btop](https://github.com/user-attachments/assets/e7d6b9a0-ce6e-49b4-9944-a923672c7b41)

**Reply by Swissisking21** [reply]
Techno Geeks
![image](https://github.com/user-attachments/assets/3abe97de-c937-4e36-a0d0-d5f96799a495)

**Comment by Shank-RTX** [comment]
OnePc
![Screenshot 2025-07-10 020656](https://github.com/user-attachments/assets/05b5dda6-6f70-43d3-9303-dd9c725ddcbc)

**Comment by Cloud-Strive** [comment]
Runtime Blinders
headnode running top, and computenode1 running htop. tmux session.

![Screenshot (8)](https://github.com/user-attachments/assets/e505693d-10ca-46ca-a7cb-9bd321693197)

**Comment by KayGee-17** [comment]
CybertronNextGen

![cybertron](https://github.com/user-attachments/assets/ca319b52-6e24-4e24-bb6d-e98c764cc871)

CybertronNextGen

**Comment by MubeenD1** [comment]
wits-1
Btop on Head node
Htop on Compute node

![image](https://github.com/user-attachments/assets/ab037e55-1ca4-4cdf-860d-e235b37c99f0)

**Comment by Kevin-Ebrahim** [comment]
![Screenshot 2025-07-09 211735](https://github.com/user-attachments/assets/83c741ba-3601-4391-b033-9903344cae64)
Wits-A

**Comment by SFBdragon** [comment]
0v1rcl0ck1d

btop running on headnode, htop on compute node

![Screenshot_20250709_225718](https://github.com/user-attachments/assets/391321ee-d61d-49fb-8e97-f0942a158b05)

**Comment by RoarkeBailey** [comment]
<img width="1918" height="1021" alt="TUT1_TMUX" src="https://github.com/user-attachments/assets/91dfd16c-7c91-4daf-9f39-8d2139132730" />

G3MIKNIGHTS running top on head node and htop on compute node

**Comment by ShavirV** [comment]
<img width="1918" height="1079" alt="image" src="https://github.com/user-attachments/assets/1ffb51d5-3330-4f71-a6d7-2dd2f9a44763" />

UPtime - `htop` running on head node and `btop` running on compute

**Comment by CadmusKei** [comment]
Team Jargon (UWC) | SCC 2026 | Tmux, Btop and Htop
Btop on top
Htop on bottom
<img width="1920" height="1080" alt="image" src="https://github.com/user-attachments/assets/32387976-ef6c-4673-aab5-e96505d0f976" />

**Comment by saxs-14** [comment]
3ncrypted (UMP) T2 Tmux, Btop and Htop
<img width="1918" height="1078" alt="image" src="https://github.com/user-attachments/assets/69a74965-65f7-401d-a169-6350e4cb9224" />

**Comment by LufunoMiles** [comment]
Digital Dynamos ( University of Venda) <img width="1827" height="962" alt="Screenshot from 2026-07-07 01-02-35" src="https://github.com/user-attachments/assets/75e5cb33-045f-4499-a395-a4c2be38a635" />

**Comment by ThapeloVundla** [comment]
DC Spears | Tutorial 2
1. Installation of btop(head node) and installation htop(compute node)

<img width="1860" height="999" alt="Screenshot from 2026-07-07 08-57-04" src="https://github.com/user-attachments/assets/b7f53f40-e9e3-4bf9-998a-cb5fdf34d66e" />

2. Running btop(head) and htop(compute)

<img width="1860" height="999" alt="Screenshot from 2026-07-07 09-00-14" src="https://github.com/user-attachments/assets/9a9f91fe-5208-48f8-b0eb-a38d2b75ab0a" />

3. Output deliverable showing btop(head) and htop (node)

<img width="1860" height="999" alt="Screenshot from 2026-07-07 09-01-20" src="https://github.com/user-attachments/assets/6351dd32-e125-4c7a-9b00-2e19e3b83d87" />

**Comment by bytedebuggers-dotcom** [comment]
Byte Debuggers (UFH)
**Installing btop ( head node | on the left ) and htop ( compute node | on the right )**
<img width="1905" height="1046" alt="Screenshot from 2026-07-07 12-24-14" src="https://github.com/user-attachments/assets/b40f9c02-8476-40d1-be0e-918f02afc4af" />

<img width="1905" height="1046" alt="Screenshot from 2026-07-07 12-27-04" src="https://github.com/user-attachments/assets/139c7816-c480-4032-87b3-0669ee94dfbd" />

**Running btop (head node) & htop (compute node)**
<img width="1905" height="1046" alt="Screenshot from 2026-07-07 12-26-10" src="https://github.com/user-attachments/assets/8ad04574-ff68-4434-b8b5-26434efa1ad7" />

**Comment by 23026641** [comment]
Dygital Dynamos(university of venda): htop(head node) is above and top (compute node) is below<img width="858" height="1536" alt="WhatsApp Image 2026-07-07 at 14 12 44 (1)" src="https://github.com/user-attachments/assets/c556b597-46be-4b82-a263-d29a8c138f25" />

**Comment by mabotenick97-cloud** [comment]
CyberOps

1. Running btop(head) and htop(compute)
<img width="1854" height="1048" alt="Screenshot from 2026-07-07 16-40-52" src="https://github.com/user-attachments/assets/6be0d3e5-751c-43da-a5b1-98591200f714" />
2. Output showing btop(head) and htop (node)
<img width="1854" height="1048" alt="Screenshot from 2026-07-07 16-34-52" src="https://github.com/user-attachments/assets/a13f7789-be37-47f8-beeb-0fc6059f84ec" />

**Comment by Slindokuhle19** [comment]
Unizulu ComputeX
<img width="1920" height="1080" alt="image" src="https://github.com/user-attachments/assets/aa9c2fa2-4481-4e9f-8802-419340178654" />

**Comment by majaha8** [comment]
<img width="1892" height="1037" alt="btop" src="https://github.com/user-attachments/assets/74710e3b-e364-4f71-8b80-3fa142a714d2" />
<img width="1892" height="1037" alt="htop" src="https://github.com/user-attachments/assets/358fee1d-8b9d-43bd-9fd8-44eb2d9fda29" />

**Reply by s225875411** [reply]
<img width="1850" height="998" alt="image (1)" src="https://github.com/user-attachments/assets/4449e024-8cbf-428a-92af-99060297462c" />
MadibazBits

**Comment by UNISA-QC-QUEENS** [comment]
unisa_qc_queens
<img width="1920" height="1080" alt="Screenshot 2026-07-07 175452" src="https://github.com/user-attachments/assets/9dae20b9-2c15-49b4-91bb-f7682929567e" />

**Comment by Lihle0512** [comment]
ChaosCompilers

<img width="1849" height="1047" alt="image" src="https://github.com/user-attachments/assets/58c277bd-3443-49e6-82c1-d54c18dbee9b" />

**Comment by K659** [comment]
Resilient Geeks (UL) ~ btop(head node) and htop(compute node)
<img width="1871" height="1015" alt="image" src="https://github.com/user-attachments/assets/5b5421ef-4ee2-40b4-8ca6-143ec9192703" />

**Comment by UNISA-QC-QUEENS** [comment]
UNISA_QC_QUEENS htop
<img width="1920" height="1080" alt="htop" src="https://github.com/user-attachments/assets/345bf3cc-9cc7-4a26-a34c-12de8a84510a" />

**Comment by switchsquad** [comment]
<img width="1918" height="1007" alt="image" src="https://github.com/user-attachments/assets/854c2c2b-d267-4885-9c6a-f318e37d4cde" />

Switch Squad (TUT) | Tmux, Btop and Htop

**Comment by ntando-memela** [comment]
WITS1_2026
<img width="1854" height="1048" alt="image" src="https://github.com/user-attachments/assets/b5f96ce3-96fd-4bcd-adec-ce154c5cdfba" />

**Comment by Leo-OB** [comment]
## DC Axes (CPUT) | SCC 2026
### Outputs of  Tmux, `Btop` and `Htop`
<img width="1915" height="1077" alt="T2_Btop Htop" src="https://github.com/user-attachments/assets/96fc19be-a0ea-4751-800e-da31f60cf5ab" />

**Comment by mpho21925** [comment]
Ampere Alliance | SCC 2026
Dispay of the the btop,htop and the top commands
<img width="1852" height="1006" alt="image" src="https://github.com/user-attachments/assets/2ef95446-e062-4a65-aed0-1c1c2152d7fa" />

**Comment by unisaqcqueens641-hue** [comment]
UNISA_QC_QUEENS tmux
<img width="1482" height="785" alt="tmux" src="https://github.com/user-attachments/assets/65cb5ba8-6a98-479d-aabd-e56fbf0c3381" />

**Comment by majaha8** [comment]
Team Nebula|SCC 2026
btop
<img width="935" height="1011" alt="btop" src="https://github.com/user-attachments/assets/a49547dd-e51d-4df9-9eb7-eb205ecdbe34" />
htop
<img width="935" height="1011" alt="htop" src="https://github.com/user-attachments/assets/e8a8cf4c-8ae5-49a8-a2cb-917f3c47bae1" />
tmux
<img width="935" height="1011" alt="tmux" src="https://github.com/user-attachments/assets/f06ef2bd-3e0d-4fd0-9893-bd397f5500ea" />

**Comment by Leeu-Fusi** [comment]
Cluster Commanders | CUT
btop
<img width="1850" height="1037" alt="Screenshot from 2026-07-10 14-34-10" src="https://github.com/user-attachments/assets/4f1883ba-f943-4fb0-901a-02ca8e96c96f" />
htop
<img width="1850" height="1037" alt="Screenshot from 2026-07-10 14-42-43" src="https://github.com/user-attachments/assets/146057ab-83cd-42c0-b5c1-25993c92ced4" />
tmux
<img width="1850" height="1037" alt="Screenshot from 2026-07-10 14-53-33" src="https://github.com/user-attachments/assets/7dcbdf50-4281-494d-9b21-9a38d27f4238" />

**Comment by MrLampies** [comment]
DC Spears Tmux btop and htop

<img width="1860" height="999" alt="Screenshot from 2026-07-07 09-01-20" src="https://github.com/user-attachments/assets/bd5d2abe-9787-46c0-a7d9-569d0b754ac3" />



<img width="1860" height="999" alt="Screenshot from 2026-07-07 09-00-14" src="https://github.com/user-attachments/assets/43200c8c-9160-46b8-94e2-0ff9ac857bde" />

**Comment by ShavirV** [comment]
UPtime Htop, Btop and top 
<img width="943" height="945" alt="image" src="https://github.com/user-attachments/assets/c8b0903f-8a98-45e4-bd35-3f9c0e793e44" />
